app/classes/Transvision/ShowResults.php

```diff
@@ -321,6 +325,11 @@ public static function resultsTable($search_object, $search_results, $page)
                 if we aren't in the 3locales view and if we have a $target_string
             */
             $transliterate = $locale2 == 'sr' && ! $extra_locale && $target_string && $target_string != '@@missing@@';
+            if ($current_repo === 'aurora' && $toolUsedByTargetLocale == 'locamotion') {
+                $edit_link = "https://mozilla.locamotion.org/{$locale2}/firefox/translate/{$entityPath}.po?search={$strings[1]}&sfields=source&sfields=target";
```

Comment:
As suggested in the original issue, let's search for the entity and `sfields=locations`.
You can store the exploded $key and refer to [0] and [1] instead of creating two variables.